test/integration/controllers/v1/UserController.test.coffee

```diff
@@ -0,0 +1,121 @@
+###*
+# UserController.test
+###
+
+request = require('supertest')
+should = require('should')
+
+###*
+# Test invalid /user/password endpoint.
+###
+
+describe 'Requests to the root (/user/password) path with missing required body parameters.', ->
+  it 'POST: Invalid password reset request with missing body parameters returns expected 422 - Unprocessable due to request validation error results.', (done) ->
```

Comment:
Long strings are difficult to read. General rule is to break line after 80 character: http://www.w3schools.com/js/js_conventions.asp
```
Line Length < 80
For readability, avoid lines longer than 80 characters.
```